ipatests/test_ipaserver/test_ldap.py

```diff
@@ -111,7 +112,7 @@ def test_Backend(self):
         # a client-only api. Then we register in the commands and objects
         # we need for the test.
         myapi = create_api(mode=None)
-        myapi.bootstrap(context='cli', in_server=True)
+        myapi.bootstrap(context='cli', in_server=True, confdir=paths.ETC_IPA)
```

Comment:
NACK

Tests use their own confdir for testing. Let's handle the env var in the test runner.